code/model/Subsite.php

```diff
@@ -648,7 +645,12 @@ public function getCMSFields()
             new HiddenField('IsSubsite', '', 1)
         );
 
-        $subsiteTabs->addExtraClass('subsite-model');
+		// If there are any themes available, add the dropdown
+		if (!empty($this->allowedThemes())) {
+			$fields->addFieldToTab('Root.Configuration', new DropdownField('Theme', $this->fieldLabel('Theme'), $this->allowedThemes(), $this->Theme), 'PageTypeBlacklistToggle');
```

Comment:
I know this is pre-existing, but could you swap for `DropdownField::create` so it's injectable?